app/src/main/java/fr/free/nrw/commons/nearby/NearbyController.java

```diff
@@ -245,6 +246,14 @@ public NearbyPlacesInfo loadAttractionsFromLocation(LatLng curLatLng, LatLng sea
 
         return baseMarkerOptions;
     }
+    /**
+     * A function which clears the shared preferences
+     * Used to fix #3202
+     */
+    public void removeSharedPrefs(JsonKvStore applicationKvStore, String PLACE_OBJECT) {
```

Comment:
This method seemed to me like an utility method and NearbyController is mainly to control fetching places from wikidata servers. Would be nice if you can move this to some approptiate utility class.